src/frameworks/foundation/ns_data.rs

```diff
@@ -7,6 +7,7 @@
 
 use super::ns_string::to_rust_string;
 use super::NSUInteger;
+use crate::environment::Environment;
```

Comment:
Please import `Environment` directly from the crate root (maybe I should change this convention but it's for historical reasons).

Reply:
Done